### PR TITLE
Updating Pivot to a function component inside React-Next and tweaks to React-Next and Fabric storybook

### PR DESCRIPTION
**Pull request by czearing**
#### Pull request checklist
- [X] Include a change request file using `$ yarn change`

#### Description of changes
-Converted `Pivot` to a function component and added it into react-next. 
-Changed panel position of add-ons to default to the bottom of the page for React-Next and Office-UI-Fabric-React.
-Added performance add-on to React-Next.

#### Focus areas to test
`Pivot`

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/fluentui/pull/12906)

### CODE REVIEW

File: packages/react-next/src/components/Pivot/Pivot.base.tsx [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+  }
+
+  return '';
+};
+
+const isPivotItem = (item: React.ReactNode): item is PivotItem => {
+  return ((item as React.ReactElement)?.type as React.ComponentType)?.name === PivotItem.name;
+};
+
+export const PivotBase: React.FunctionComponent<IPivotProps> = (props: IPivotProps) => {
```
Comment (xugao) [inline packages/react-next/src/components/Pivot/Pivot.base.tsx:77]: forwardRef + set displayName

**Comment by czearing** [comment]
@xugao I just created a PR for copying `Pivot` over to next.

File: change/@fluentui-react-next-2020-04-28-14-01-59-feat-pivot.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+  "type": "prerelease",
+  "comment": "Updating pivot to a function component",
```
Comment (khmakoto) [inline change/@fluentui-react-next-2020-04-28-14-01-59-feat-pivot.json:3]: nit:
```suggestion
  "comment": "Pivot: Updating component from a class component to a function component.",
```

File: change/office-ui-fabric-react-2020-04-28-14-01-59-feat-pivot.json [hunk 3]
```diff
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
+  "comment": "Changing storybook addon panel position",
```
Comment (khmakoto) [inline change/office-ui-fabric-react-2020-04-28-14-01-59-feat-pivot.json:3]: nit:
```suggestion
  "comment": "Changing storybook add-on panel position to default to be at the bottom of the page.",
```

File: packages/react-next/src/components/Checkbox/Checkbox.test.tsx [hunk 4]
```diff
@@ -3,7 +3,7 @@ import * as renderer from 'react-test-renderer';
 import { mount, ReactWrapper } from 'enzyme';
 
 import { Checkbox } from './Checkbox';
-import { IRefObject, resetIds } from '../../Utilities';
+import { IRefObject, resetIds } from '../..//Utilities';
```
Comment (khmakoto) [inline packages/react-next/src/components/Checkbox/Checkbox.test.tsx:6]:
```suggestion
import { IRefObject, resetIds } from '../../Utilities';
```

File: packages/office-ui-fabric-react/src/components/__snapshots__/Pivot.SeparateNoSelectedKey.Example.tsx.shot [hunk 5]
```diff
@@ -31,511 +31,524 @@ exports[`Component Examples renders Pivot.SeparateNoSelectedKey.Example.tsx corr
       <div
```
Comment (khmakoto) [inline packages/office-ui-fabric-react/src/components/__snapshots__/Pivot.SeparateNoSelectedKey.Example.tsx.shot:31]: Why is this snapshot being updated?

Comment (czearing) [reply packages/office-ui-fabric-react/src/components/__snapshots__/Pivot.SeparateNoSelectedKey.Example.tsx.shot]: Not entirely sure why that happend. I pulled from upstream and reverted the snapshot update and it works now.

**Pull request merged**